spec/client_spec.rb

```diff
@@ -12,8 +12,10 @@
   end
 
   it "should #authorize method call" do
-    res = client.authorize
-    expect(res.status).to eq 200
+    stub_request(:post, "https://test.amocrm.ru/private/api/auth.php?type=json").
+        with(:body => "{\"USER_LOGIN\":\"\",\"USER_HASH\":\"\"}",
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax.<br>Style/MultilineMethodCallIndentation: Use 2 (not 4) spaces for indenting an expression spanning multiple lines.